sdk/keyvault/azkeys/models.go

```diff
@@ -0,0 +1,393 @@
+//go:build go1.16
+// +build go1.16
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+package azkeys
+
+import (
+	"time"
+
+	generated "github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys/internal/generated"
+)
+
+// Attributes - The object attributes managed by the KeyVault service.
+type Attributes struct {
+	// Determines whether the object is enabled.
+	Enabled *bool `json:"enabled,omitempty"`
+
+	// Expiry date in UTC.
+	Expires *time.Time `json:"exp,omitempty"`
+
+	// Not before date in UTC.
+	NotBefore *time.Time `json:"nbf,omitempty"`
+
+	// READ-ONLY; Creation time in UTC.
+	Created *time.Time `json:"created,omitempty" azure:"ro"`
+
+	// READ-ONLY; Last updated time in UTC.
+	Updated *time.Time `json:"updated,omitempty" azure:"ro"`
+}
+
+// KeyAttributes - The attributes of a key managed by the key vault service.
+type KeyAttributes struct {
+	Attributes
+	// READ-ONLY; softDelete data retention days. Value should be >=7 and <=90 when softDelete enabled, otherwise 0.
```

Comment:
soft delete is always enabled
```suggestion
	// READ-ONLY; softDelete data retention days. Value should be >=7 and <=90
```
But if this is read only, is it useful to document the range of valid values?

Reply:
I pulled this documentation from the service, but I agree it probably doesn't need to be there because it's read only